scripts/GetMLPythonSamplesOffGithub.ps1

```diff
@@ -0,0 +1,6 @@
+
+write-host "Downloading Python ML samples to your desktop ..."
+
+Update-SessionEnvironment
+cd $env:USERPROFILE\desktop
+git clone https://github.com/Microsoft/Dev-Advocacy-Samples/tree/master/python-mlclassification
```

Comment:
Not sure we need to clone the repo since the tutorial is mainly a set of commands to use in the Ubuntu terminal in the README

Reply:
but you get the iris tutorial right - isn't that helpful?

Reply:
Yes it's helpful for someone who wants the end classification solution rather than the whole walkthrough (run the .py script)